gis/gccview/gcc_layers_pull.py

```diff
@@ -0,0 +1,130 @@
+# The official new GCC puller DAG file
+import sys
+import os
+repo_path = os.path.abspath(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+sys.path.insert(0,os.path.join(repo_path,'gis/gccview'))
+from gcc_layer_refresh import get_layer
+
+from datetime import datetime
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+
+# Credentials - to be passed through PythonOperator
+from airflow.hooks.postgres_hook import PostgresHook
+# EC2 connection credentials
+ec2_cred = PostgresHook("gcc_bot_bigdata")
+# Morbius connection credentials
+morbius_cred = PostgresHook("gcc_bot")
+
+SLACK_CONN_ID = 'slack'
+def task_fail_slack_alert(context):
+    slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
+    slack_msg = """
+            :red_circle: Task Failed. 
+            *Task*: {task}  
+            *Dag*: {dag} 
+            *Execution Time*: {exec_date}  
+            *Log Url*: {log_url} 
+            """.format(
+            task=context.get('task_instance').task_id,
+            dag=context.get('task_instance').dag_id,
+            ti=context.get('task_instance'),
+            exec_date=context.get('execution_date'),
+            log_url=context.get('task_instance').log_url,
+        )
+    failed_alert = SlackWebhookOperator(
+        task_id='slack_test',
+        http_conn_id='slack',
+        webhook_token=slack_webhook_token,
+        message=slack_msg,
+        username='airflow',
+        proxy='http://137.15.73.132:8080')
```

Comment:
Can you update the proxy to be `'http://'+BaseHook.get_connection('slack').password+'@137.15.73.132:8080'` to contain username and password?